Sito/css/style.css

```diff
@@ -213,8 +213,10 @@ body a:visited {
 }
 
 #footer p {
+	color: #d9d9d9;
   font-size: 0.8em;
   margin-bottom: 0;
+	padding: 1em;
```

Comment:
Questo paddin si applica ad ogni p del footer che diventa così nel lato utente che ha 3 p
![image](https://user-images.githubusercontent.com/56229661/69882862-f2408180-12d1-11ea-83e2-3c284692424c.png)
Mi sembra un po' troppo separato. Tu lo hai messo perchè ti serve nel lato admin. Prova ad usare uan cosa del genere `.info:nth-child(1)` però applicata al p così si applica solo al primo e non facciamo chilometri di padding. Un'altra cosa il padding applicalo solo sopra e sotto se non serve applicarlo tutto in torno. Qui non mi sembra serva.

Reply:
Ora ho fatto prima inizializzato padding e margin a 0, e poi gli ho dato quel poco che basta, mi sembra più compatto.